_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png).
developing Image and website system.